setup.py

```diff
@@ -1,18 +1,27 @@
 from setuptools import setup
 
+with open('README.md', 'r') as f:
```

Comment:
Nope. I will not allow this. setup.py being a script is not something I'm okay with.

Reply:
Please explain

Reply:
Unlike other language's manifests, setup.py is a python script. The current one this this package is entirely declarative (i.e. it could be edited with regexes, etc with no problem). However, adding reading from a file would make it no so anymore. I'm against the idea of this. I'll probably convert this to use setup.cfg in the future anyway.

Reply:
How do we make README.md's contents show up on PyPI then?

Reply:
setup.cfg supports referring to a file: https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files I can make this change when I update to use setup.cfg.